vnext/Microsoft.ReactNative/Fabric/ViewComponentView.cpp

```diff
@@ -47,27 +48,36 @@ void ViewComponentView::updateProps(
     auto color = *newViewProps.backgroundColor;
 
     if (newViewProps.backgroundColor) {
-      m_element.ViewBackground(SolidColorBrushFrom(newViewProps.backgroundColor));
+      m_panel.ViewBackground(SolidColorBrushFrom(newViewProps.backgroundColor));
     } else {
-      m_element.ClearValue(winrt::Microsoft::ReactNative::ViewPanel::ViewBackgroundProperty());
+      m_panel.ClearValue(winrt::Microsoft::ReactNative::ViewPanel::ViewBackgroundProperty());
     }
   }
 
   if (oldViewProps.borderColors != newViewProps.borderColors) {
     if (newViewProps.borderColors.all) {
-      m_element.BorderBrush(SolidColorBrushFrom(*newViewProps.borderColors.all));
+      m_panel.BorderBrush(SolidColorBrushFrom(*newViewProps.borderColors.all));
     } else {
-      m_element.ClearValue(winrt::Microsoft::ReactNative::ViewPanel::BorderBrushProperty());
+      m_panel.ClearValue(winrt::Microsoft::ReactNative::ViewPanel::BorderBrushProperty());
     }
   }
 
-  if (oldViewProps.borderStyles != newViewProps.borderStyles) {
+  if (oldViewProps.opacity != newViewProps.opacity) {
+    m_panel.Opacity(newViewProps.opacity);
+  }
+
+  if (oldViewProps.borderStyles != newViewProps.borderStyles || oldViewProps.borderRadii != newViewProps.borderRadii) {
     m_needsBorderUpdate = true;
   }
 
   m_props = std::static_pointer_cast<facebook::react::ViewProps const>(props);
 }
 
+bool ViewComponentView::shouldBeControl() const noexcept {
+  // Fabric does not appear to have the focusable prop right now...
+  return false; // m_props.focusable || HasDynamicAutomationProperties(view);
```

Comment:
Curious of the history of this. `View` had a handwritten ViewConfig right? Is it just missing this?

Might be worth filing an issue to follow up on either way.

Reply:
View, and most of the very fundamental components are all still hand written. (View, Image, TextInput, ScrollView are all hand written)